applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -283,12 +388,23 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 		     BT_AUDIO_EP_STATE_CODEC_CONFIGURED))
 #endif
 		{
+			if (headsets[channel_index].pd != pres_delay) {
+				audio_play_pause(false);
```

Comment:
I think we may be better off splitting into `audio_play `and `audio_pause`.

Reply:
True, I needed a stop function so as to ensure the change occurs for the new pres delay. Would prefer separate play and pause functions. It would also be good to have a stop function as pause implies the audio will continue from where it was paused, which may not always be true (e.g. live streams, gaming, broadcasts). Stop is a clear that we will get no more audio from that source.
I have a global playing_state to test against, which I have missed the test for!! Will add the test of playing_state to decide if the stream needs to pause and restart.